fetch_teleop/src/joystick_teleop.cpp

```diff
@@ -380,7 +379,8 @@ class GripperTeleop : public TeleopComponent
     ros::NodeHandle pnh(nh, name);
 
     // Button mapping
-    pnh.param("button_deadman", deadman_, 10);
+    pnh.param("button_deadman", deadman_body_, 9);
```

Comment:
It is fine to add additional deadman buttons (for instance, I think you want 3 deadmen, one for the body, one for ee frame, one for body frame -- basically saying regardless of what you are controlling, it's OK to enable the gripper buttons)